nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/SchemaManagerImpl.java

```diff
@@ -848,4 +856,54 @@ public boolean getClearComplexPropertyBeforeSet() {
         return clearComplexPropertyBeforeSet;
     }
 
+    /*
+     * ===== Property API =====
+     */
+
+    /**
+     * @since 11.1
+     */
+    protected synchronized void registerSecuredProperty(PropertyDescriptor descriptor) {
+        allSecuredProperties.add(descriptor);
+    }
+
+    /**
+     * @since HF
+     */
+    protected synchronized void unregisterSecuredProperty(PropertyDescriptor descriptor) {
+        allSecuredProperties.remove(descriptor);
+    }
+
+    protected void recomputeSecuredProperties() {
```

Comment:
Curious why is it not needed in upper versions?

Reply:
Because it's handled by the default descriptor registry since 10.10.